### PR TITLE
Bug fixes to make Slack voting work

### PR DESCRIPTION
**Pull request by mashton**
A bunch of bug fixes to make the following policy work (Adapted from StarterKit):

```
                                       initialize = """
usernames = [u.username for u in users]
jury = random.sample(usernames, k=3)
action.data.set('jury', jury)
                                           """,
                                       check = """
import datetime
jury = action.data.get('jury')
jury_users = users.filter(username__in=jury)
yes_votes = action.proposal.get_yes_votes(users=jury_users)
if len(yes_votes) >= 2:
   return PASSED
elif action.proposal.time_elapsed() > datetime.timedelta(days=2):
   return FAILED
                                           """,
                                       notify = """
jury = action.data.get('jury')
jury_users = users.filter(username__in=jury)
action.community.notify_action(action, policy, users=jury_users, template='Please deliberate amongst yourselves before voting')
                                           """,
```

### CODE REVIEW

File: policykit/policyengine/models.py [hunk 1]
```diff
@@ -775,8 +780,12 @@ def save(self, *args, **kwargs):
         logger.info('entered save')
         if not self.pk:
             logger.info('is pk')
+            if self.data is None:
+                ds = DataStore.objects.create()
+                self.data = ds
```
Comment (mashton) [inline policykit/policyengine/models.py:785]: Is this the right place to initialize this @rickygv99? I was looking at how the `data` datastore is created for Proposals [here](https://github.com/amyxzhang/policykit/blob/c6bcb88240d1bd79e8b01743222ab2fd96883dac/policykit/policyengine/models.py#L242-L244).

**Comment by mashton** [comment]
@amyxzhang @rickygv99 this is ready for a look. I can update all the rest of the starterkit examples and the [documentation](https://policykit.metagov.org/main/documentation) once this goes in, so they both can match the actual functionality.

**Comment by rickygv99** [comment]
Looks good to me!

File: policykit/integrations/slack/models.py [hunk 2]
```diff
@@ -35,20 +35,16 @@ class SlackCommunity(Community):
     bot_id = models.CharField('bot_id', max_length=150, unique=True, default='')
 
     def notify_action(self, action, policy, users, post_type='channel', template=None, channel=None):
-        from slackintegration.views import post_policy
+        from intergrations.slack.views import post_policy
```
Comment (amyxzhang) [inline policykit/integrations/slack/models.py:38]: just checking is this a typo? Shouldn't it be "integrations"

Comment (mashton) [reply policykit/integrations/slack/models.py:38]: 🤦‍♀️  thank you

**Pull request merged**